src/interfaces/court-order-interface.ts

```diff
@@ -2,8 +2,9 @@
  * See:
  * https://github.com/bcgov/business-schemas/blob/master/src/registry_schemas/schemas/court_order.json
  */
-export interface CourtOrderIF{
+export interface CourtOrderIF {
   fileNumber: string
   orderDate: string
-  effectOfOrder: string
+  effectOfOrder: string,
```

Comment:
If you're in here again, remove the trailing comment. It's not needed and generally our code style is to leave it out so it looks cleaner.

Reply:
Whoops, good eye